api/services/ocean/aws/schemas/ocean-compute.yaml

```diff
@@ -45,13 +51,15 @@ properties:
         type: string
         description: ID of the image used to launch the instances.
       userData:
-        type: base64
```

Comment:
wrong change.
you are welcome to un locally and see the difference

Reply:
Added format for base64. The data type of base64 does not exist.